crates/burrego/.github/workflows/tests.yml

```diff
@@ -0,0 +1,77 @@
+on: [push, pull_request]
```

Comment:
I wonder if this GitHub action is ran, that would be nice

Reply:
Let me check. If not, I can try some tricks like a symlink. I will report back.

Reply:
A symlink does not work. We will have to have a targeted workflow for it in the root folder.

Reply:
Opened https://github.com/kubewarden/policy-evaluator/issues/18